### DISCUSSION TITLE
Synology: How to keep logs after Watchtower has updated the container?

### DISCUSSION THREAD

**Discussion by mtoensing** [containrrr/watchtower#609] [Q&A] [answered]
Hi! 

Watchtower runs great. It keeps all my containers up-to-date. But sadly all my logs are wiped after each update. Is there a way to keep the logs of an updated container? 

<img width="1440" alt="Screen Shot 2020-07-29 at 13 10 17" src="https://user-images.githubusercontent.com/136737/88793331-e8db2200-d19c-11ea-9dc2-68acc33f64ed.png">

**Comment by zoispag** [comment, marked as answer]
Not sure how this would affect synology logs, but maybe you can use a different logging driver.

From docs:

> When using Docker Community Engine, the docker logs command is only available on the following drivers:
> - local
> - json-file
> - journald

As long as docker logs work, I suspect that Synology GUI will fetch them. So you need to find a way to persist the file, between container re-creation.

**Reply by simonaronsson** [reply]
[Here](https://stackoverflow.com/a/43663253/1511964) is a stackoverflow answer describing how to do this using journald and tags. This will likely lead to the logs not being searchable through the synology UI, just as @zoispag states, but other than that, you should be good to go.
